app/models/admin_user.rb

```diff
@@ -32,8 +32,10 @@ class AdminUser < ActiveRecord::Base
   validates_format_of :email, with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i, unless:  proc { self.persisted? && email.blank? }
 
   after_save do
-    contact = billing_contact || build_billing_contact
-    contact.update!(email: self.email) if @email
+    unless Rails.env.test?
```

Comment:
@Fivell All schemas except "public" do not exist in DB when specs runing. We are using PG as a docker container with this image https://hub.docker.com/r/maxkonin/yeti_pg/. Mb something wrong with configs.... But I think that all be fine after 85 PR